portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,45 @@
+
+package com.google.sps.servlets;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/comments")
+public class CommentServlet extends HttpServlet {
+
+    private CommentSection section = new CommentSection();
```

Comment:
Where is this class defined?

Reply:
Resolving because I ended up deleting this class